### PR TITLE
updtate data for summer term 2015 (as from 2015-02-18

### PR DESCRIPTION
**Pull request by sebastiankliem**

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -90,6 +90,18 @@ var data = {
         dozent: ["Dr. Gjergji Kasneci"],
         nameLV: "Advanced Recommendation Techniques"
     },
+    advancaedSoftwareEngineeringForEmbeddedSystems:
```
Comment (knub) [inline js/data.js]: typo

File: js/data.js [hunk 2]
```diff
@@ -139,7 +151,7 @@ var data = {
         kennung: [SSKSK, SSKKO],
         lehrform: ["Seminar"],
         modul: ["Softskills"],
-        semester: [ss13, ws13_14, ss14, ws14_15],
+        semester: [ss13, ws13_14, ss14, ws14_15, ss15],
```
Comment (knub) [inline js/data.js]: Can't find that seminar for the next semester.

File: js/data.js [hunk 3]
```diff
@@ -227,7 +239,7 @@ var data = {
         kennung: [OSIS, SAMT],
         lehrform: ["Vorlesung", "Übung"],
         modul: ["Vertiefungsgebiete"],
-        semester: [ws13_14],
+        semester: [ws13_14, ss15],
         benotet: true,
         cp: 6,
         dozent: ["Dr. Gjergji Kasneci"],
```
Comment (knub) [inline js/data.js]: Different lecturers this time.

File: js/data.js [hunk 4]
```diff
@@ -244,6 +256,18 @@ var data = {
         dozent: ["Prof. Dr. Felix Naumann" , "Thorsten Papenbrock"],
         nameLV: "Data Profiling and Data Cleansing"
     },
+    decisionManagement:
+    {
+      kurz: "Decision<br>Management",
+      kennung: [BPET, OSIS],
+      lehrform: ["Seminar"],
+      modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
+      semester: [ss13, ss14],
```
Comment (knub) [inline js/data.js]: Semesters are wrong.

File: js/data.js [hunk 5]
```diff
@@ -271,12 +295,24 @@ var data = {
...
-        semester: [ss13,ws13_14, ss14, ws14_15],
+        semester: [ss13,ws13_14, ss14, ws14_15, ss15],
         benotet: true,
         cp: 9,
         dozent: ["Dr. Claudia Nicolai", "Prof. Ulrich Weinberg"],
         nameLV: "Design Thinking Basics (D-School Basic Track)"
     },
+    distributedBigDataAnalytics:
+    {
+      kurz: "Bid Data<b>Analytics",
```
Comment (knub) [inline js/data.js]: Typo

File: js/data.js [hunk 6]
```diff
@@ -271,12 +295,24 @@ var data = {
...
+      kennung: [OSIS, BPET],
+      lehrform: ["Seminar"],
+      modul: ["IT Systems Engineering", "Vertiefungsgebiete"],
```
Comment (knub) [inline js/data.js]: "IT Systems Engineering" vs "IT-Systems Engineering".

File: js/data.js [hunk 7]
```diff
@@ -315,7 +351,7 @@ var data = {
         kennung: [ITSE, OSIS],
         lehrform: ["Vorlesung"],
         modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
-        semester: [ss14],
+        semester: [ss14, ss15],
         benotet: true,
         cp: 6,
```
Comment (knub) [inline js/data.js]: Different lecture, because it's only three points now. Be careful.

**Comment by knub** [comment]
Hi,

I left some comments in the code. Please be careful when editing, especially when "Vertiefungsgebiete" or credit points change, this will turn out bad if people rely on it.

File: js/data.js [hunk 8]
```diff
@@ -530,6 +610,17 @@ var data = {
...
+    inMemoryHealthcare: {
+        kurz: "IMDB<br>Healthcare",
+        kennung: [ITSE, OSIS, BPET],
+        lehrform: ["Seminar"],
+        modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
+        semester: [ss15],
+        benotet: true,
+        cp: 6,
+        dozent: ["Dr. Matthias Uflacker", "Dr. Matthieu Schapranow"],
+        nameLV: "DIn-Memory Databases: Applications in Healthcare"
```
Comment (knub) [inline js/data.js]: typo

File: js/data.js [hunk 9]
```diff
@@ -645,7 +747,7 @@ var data = {
         kennung: [OSIS, BPET],
         lehrform: ["Vorlesung", "Übung"],
         modul: ["Vertiefungsgebiete"],
-        semester: [ss14],
+        semester: [ss14, ss15],
         benotet: true,
         cp: 3,
         dozent: ["Dr. Mariana Neves", "Prof. Dr. Felix Naumann"],
```
Comment (knub) [inline js/data.js]: Remove Naumann.

File: js/data.js [hunk 10]
```diff
@@ -739,6 +863,28 @@ var data = {
...
         nameLV: "Non-Uniform Memory Access (NUMA) - Herausforderungen paralleler Systeme "
     },
+    parProg: {
+        kurz: "Parallele<br>Systeme",
+        kennung: [SAMT, OSIS, IST],
+        lehrform: ["Vorlesung"],
+        modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
+        semester: [ss15],
+        benotet: true,
+        cp: 6,
```
Comment (knub) [inline js/data.js]: That's only 3 points. Be careful.

File: js/data.js [hunk 11]
```diff
@@ -739,6 +863,28 @@ var data = {
...
+        dozent: ["Prof. Dr. Andreas Polze"],
+        nameLV: "Programmierung paralleler und verteilter Systeme"
+    },
+    parProgInMemory: {
+        kurz: "Parallel Prog.<br>In Memory",
+        kennung: [SAMT, OSIS, BPET],
```
Comment (knub) [inline js/data.js]: ITSE missing

File: js/data.js [hunk 12]
```diff
@@ -908,7 +1065,7 @@ var data = {
         kennung: [IST, OSIS, HCT],
```
Comment (knub) [inline js/data.js]: OSIS BPET IST

**Comment by philippotto** [comment]
Any chance this PR will be finalized soon, @sebastiankliem?

**Comment by knub** [comment]
Hi Sebastian,

I am sorry, but I have to close this, as it is now unmergeable due to #20.
Still, thank you for your work.

**Pull request closed without merging**